docs/usage/installation.rst

```diff
@@ -1,2 +1,28 @@
-SCAutolib installation!
+SCAutolib Setup
 ========================
+
+In order to be able to prepare the system for smart cards testing via the SCAutolib, following RPM packages are required:
+
+* httpd
+* opensc
+* openssl
+* sssd
+* gnutls-utils
+* gdm (optional, installed if ``--gdm`` switch is used)
+
+If virtual smart card type is used for one of the users specified in configuration file, following additional packages are required:
+
+* pcsc-lite-ccid
+* pcsc-lite
+* virt_cacard
+* vpcd
+* softhsm
+
+Those packages can be installed using library by specifying ``--install-missing`` option to ``scauto prepare`` command.
+However, you need to provide repository for these packages.
+Most of the packages can be installed from BaseOS and AppStream repositories, but some packages requires additional repositories.
```

Comment:
BaseOS and AppStream are RHEL specific while library is upstream. If we give this information, we should mention it is RHEL, for example:

On RedHat linux, most of these packages...